build/gulp/js-bundles.js

```diff
@@ -57,23 +62,28 @@ gulp.task('js-bundles-prod', gulp.series('version-replace', function() {
         .pipe(gulp.dest(context.RESULT_JS_PATH));
 }));
 
-
-const createDebugBundlesStream = function(watch) {
+function prepareDebugMeta(watch) {
     let debugConfig;
     let bundles;
     if(watch) {
         debugConfig = Object.assign({}, webpackConfigDev);
-        bundles = processDevBundles(DEBUG_BUNDLES);
+        bundles = processBundles(DEBUG_BUNDLES, 'js');
     } else {
         debugConfig = Object.assign({}, webpackConfig);
-        bundles = processBundles(DEBUG_BUNDLES);
+        bundles = processBundles(DEBUG_BUNDLES, context.TRANSPILED_PROD_PATH);
     }
     debugConfig.output = Object.assign({}, webpackConfig.output);
     debugConfig.output['pathinfo'] = true;
     if(!context.uglify) {
         debugConfig.devtool = 'eval-source-map';
     }
 
+    return { debugConfig, bundles };
+}
+
+function createDebugBundlesStream(watch) {
```

Comment:
I have noticed that createDebugBundlesStreamRenovation similar to createDebugBundlesStream. The difference is in the destination. Maybe we can merge these functions?